examples/openapi-tools/quickstart-se/README.md

```diff
@@ -0,0 +1,352 @@
+        String name = request.path().param("name");
+        api.getMessage(name)
+           .webClientResponse()
+           .flatMapSingle(serverResponse -> serverResponse.content().as(Message.class))
+           .thenAccept(response::send);
+    }
+
+    /**
+     * PUT /greet/greeting : Set the greeting to use in future messages..
+     *
+     * @param request  the server request
+     * @param response the server response
+     * @param message  Message for the user
+     */
+    public void updateGreeting(ServerRequest request, ServerResponse response, Message message) {
+        api.updateGreeting(message)
+           .webClientResponse()
+           .thenAccept(content -> response.status(Http.Status.NO_CONTENT_204).send());
+    }
+}
+```
+
+3) Add class `Main` that will be the main class for our client :
+```java
+public final class Main {
+
+    /**
+     * Cannot be instantiated.
+     */
+    private Main() {
+    }
+
+    /**
+     * Application main entry point.
+     * @param args command line arguments.
+     */
+    public static void main(final String[] args) {
+        startServer();
+    }
+
+    /**
+     * Start the server.
+     * @return the created {@link WebServer} instance
+     */
+    static Single<WebServer> startServer() {
+
+        // load logging configuration
+        LogConfig.configureRuntime();
+
+        // By default this will pick up application.yaml from the classpath
+        Config config = Config.create();
+
+        WebServer server = WebServer.builder(createRouting(config))
+                                    .config(config.get("server"))
+                                    .addMediaSupport(JacksonSupport.create())
+                                    .build();
+
+        Single<WebServer> webserver = server.start();
+
+        // Try to start the server. If successful, print some info and arrange to
+        // print a message at shutdown. If unsuccessful, print the exception.
+        webserver.thenAccept(ws -> {
+                     System.out.println("WEB server is up! http://localhost:8081");
+                     ws.whenShutdown().thenRun(() -> System.out.println("WEB server is DOWN. Good bye!"));
+                 })
+                 .exceptionallyAccept(t -> {
+                     System.err.println("Startup failed: " + t.getMessage());
+                     t.printStackTrace(System.err);
+                 });
+
+        return webserver;
+    }
+
+    /**
+     * Creates new {@link Routing}.
+     *
+     * @return routing configured with JSON support, a health check, and a service
+     * @param config configuration of this server
+     */
+    private static Routing createRouting(Config config) {
+
+        return Routing.builder()
+                      .register("/", new MessageService())
+                      .build();
+    }
+}
+```
+
+4) Create `resource` folder and put a `application.yaml` file inside.
```

Comment:
Change to 

Create the directory `src/main/resources/`. Create `application.yaml` in that directory with the following content: